src/app.js

```diff
@@ -3,21 +3,43 @@ require('dotenv').config();
 const express = require('express');
 const logger = require('morgan');
 const bodyParser = require('body-parser');
+const passport = require('passport');
+const KeycloakBearerStrategy = require('passport-keycloak-bearer');
 
 const fhirRouter = require('./routes/fhir');
 const indexRouter = require('./routes/index');
 const serversRouter = require('./routes/servers');
+const wellKnownRouter = require('./routes/wellknown');
 const subscriptionsRouter = require('./routes/subscriptions');
 
+const { backendAuthorization, subscriptionAuthorization } = require('./utils/auth');
+
 const app = express();
 
 app.use(logger('dev'));
 app.use(express.json());
 app.use(bodyParser.json({ type: ['application/json', 'application/fhir+json'] }));
+app.use(passport.initialize());
+
+passport.use(
+  new KeycloakBearerStrategy(
+    {
+      realm: 'backend_app',
+      url: 'http://moonshot-dev.mitre.org:8090/auth'
```

Comment:
Can this url and the token endpoint in `src/routes/wellknown.js` be made configurable? Those would work well in a .env file. Maybe we should also have a .env.dev or .env.mitre or something with these pre-populated.

Reply:
Added REALM and AUTH env vars which are required. Pushed .env.dev which contains an example .env file